cookbook/smart-content-data-provider.rst

```diff
@@ -0,0 +1,48 @@
+DataProvider for SmartContent
+=============================
+
+DataProviders are used to load data for SmartContent. It returns
+data filtered by a configuration array. This array can be
+configured with an overlay in the backend form.
+
+This filters includes following values:
+
+.. list-table::
+    :header-rows: 1
+
+    * - Name
+      - Description
+    * - DataSource
+      - Additional constraint - like page-"folder"
+    * - Tags
+      - Multiple selection of tags, which a item should have - all or one of them
```

Comment:
Shouldn't all the values from the configuration overlay be listed here? `All or none of them` is a separate dropdown, isn't it? Apart from that I am missing the limit, and both sorting dropdowns.